layers/meta-balena-raspberrypi/recipes-core/resin-extra-udev-rules/resin-extra-udev-rules.bbappend

```diff
@@ -0,0 +1,10 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
+
+SRC_URI = " \
+	file://86-nm-unmanaged-fin.rules \
+	"
+
+do_install_append() {
```

Comment:
The above should really be `do_install_append_fincm3() {` so that we only do this for the Balena Fin